src/extend-context-menu.ts

```diff
@@ -3,9 +3,9 @@ import { onContextMenu } from "@nebula.js/stardust";
 import copyCellValue from "./table/utils/copy-utils";
 
 export default function extendContextMenu() {
-  onContextMenu?.((menu: any, event: any) => {
-    event.target &&
-      event.target.closest(".sn-table-cell") &&
+  onContextMenu?.((menu: any, event: Event) => {
```

Comment:
Pretty sure this is some kind of event, could be both keyboard and mouse. So using a general type here